qemu-cc/qemu-cc.spec

```diff
@@ -0,0 +1,214 @@
+Name: qemu-cc
+Version: v2.9.0+git.a2e6957b6f
+Release: 0
+Source0: %{name}-%{version}.tar.gz
+Source1: qemu-cc-rpmlintrc
+BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}
+
+Summary  : OpenBIOS development utilities
+Group    : Development/Tools
+License  : BSD-2-Clause BSD-3-Clause GPL-2.0 GPL-2.0+ LGPL-2.0+ LGPL-2.1
+
+Requires: qemu-cc-bin
+Requires: qemu-cc-data
+
+BuildRequires : automake
+BuildRequires : bison
+BuildRequires : flex
+BuildRequires : glib2-devel
+BuildRequires : libattr-devel
+BuildRequires : libcap-devel
+BuildRequires : libcap-ng-devel
+BuildRequires : libtool
+BuildRequires : libtool-ltdl-devel
+BuildRequires : libtool
+BuildRequires : m4
+BuildRequires : numactl-devel
+BuildRequires : python-devel
+BuildRequires : zlib
```

Comment:
The `qemu-lite.spec` specifies this as:

```
BuildRequires : zlib-devel
```

... which seems "more correct" since that will pull in the `zlib` package.